google-datacatalog-rdbms-connector/tests/google/datacatalog_connectors/rdbms/prepare/sql_objects/datacatalog_sql_objects_tag_template_factory_test.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/python
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import unittest
+
+from google.datacatalog_connectors.commons_test import utils
+from google.datacatalog_connectors.rdbms.prepare.sql_objects import \
+    datacatalog_sql_objects_tag_template_factory
+from google.cloud import datacatalog
```

Comment:
Please import this module before `google.datacatalog_connectors.` to keep the standards.